### PR TITLE
[All hosts] (auth) clarify SSO support

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Also fixes this issue in another repo: https://github.com/OfficeDev/Office-Add-in-NodeJS-SSO/issues/26

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[cc152d0](https://github.com/OfficeDev/office-js-docs-pr/pull/761/commits/cc152d0a63193b8ac40d3592fc950e8d538c1be5)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[cc152d0](https://github.com/OfficeDev/office-js-docs-pr/pull/761/commits/cc152d0a63193b8ac40d3592fc950e8d538c1be5)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[cc152d0](https://github.com/OfficeDev/office-js-docs-pr/pull/761/commits/cc152d0a63193b8ac40d3592fc950e8d538c1be5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/sso-support/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902182203567655-761%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FBS7FKN%2FSuKoqnLQYjuKV2zUTaz1RSIVcTULBlRZId4%3D&st=2019-02-18T22%3A00%3A12Z&se=2019-03-21T22%3A05%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/identity-api-requirement-sets.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
 
 Office Add-ins run across multiple versions of Office. The following table lists the Identity API requirement sets, the Office host applications that support that requirement set, and the build or version numbers for the Office application.
 
-|  Requirement set  | Office 2013 for Windows | Office 365 for Windows   |  Office 365 for iPad  |  Office 365 for Mac  | Office Online  | SharePoint Online | OneDrive.com |Outlook.com & Exchange Online|
+|  Requirement set  | Office 2013/16/19 for Windows | Office 365 for Windows   |  Office 365 for iPad  |  Office 365 for Mac  | Office Online  | SharePoint Online | OneDrive.com |Outlook.com & Exchange Online|
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/identity-api-requirement-sets.md]: Maybe we want to be explicit that this is 2016 and 2019. I'll admit I read this as a date first, then realized it couldn't be and thought it was a version number until it clicked.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[de5af47](https://github.com/OfficeDev/office-js-docs-pr/pull/761/commits/de5af47d5200143db794a116777827d2c11a6a2e)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[de5af47](https://github.com/OfficeDev/office-js-docs-pr/pull/761/commits/de5af47d5200143db794a116777827d2c11a6a2e)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[de5af47](https://github.com/OfficeDev/office-js-docs-pr/pull/761/commits/de5af47d5200143db794a116777827d2c11a6a2e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/requirement-sets/identity-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/sso-support/docs/reference/requirement-sets/identity-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902191839497994-761%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=TEx05X2I1NNeTESr%2FDUeRw2tNyFRKCv4dbKri0OTtFc%3D&st=2019-02-19T18%3A36%3A09Z&se=2019-03-22T18%3A41%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Rick-Kirkham** [comment]
@AlexJerabek I've put this question on the agenda for the next team meeting.

**Pull request merged**